Source/WTF/wtf/ThreadSafeRefCounted.h

```diff
@@ -107,6 +107,12 @@ class ThreadSafeRefCountedBase {
 private:
     mutable std::atomic<unsigned> m_refCount { 1 };
 
+#if ASSERT_ENABLED
+    // Match the layout of RefCounted, which has flag bits for threading checks.
+    WK_UNUSED_INSTANCE_VARIABLE bool m_unused1;
```

Comment:
Not sure if this is the best way to fix. My preference would be:
a. Disable the size assertions on 32 bit
or
b. Generate a bit better assertions to differentiate ThreadSafeRefCounted and RefCounted (a bit harder)

@achristensen07 what do you think?

Reply:
I think we want the size assertions on all platforms because we have some members of some structures that are platform-specific.  I think annotating [ThreadSafeRefCounted] and [RefCounted] and keeping them 100% accurate for all 150+ serialized refcounted types would be an unnecessary burden just to reduce the size of the structures in debug builds.  I really think this is the best solution.

It would be helpful to add a static_assert like I have in https://github.com/WebKit/WebKit/pull/33223 though.  Since this PR was first, I'll close mine.